src/app/blocks/web-money/web-money.component.ts

```diff
@@ -271,9 +271,26 @@ export class WebMoneyComponent extends BaseBlockComponent {
       }
 
     }
-    if (document['monetization']) {
-      document['monetization'].addEventListener('monetizationprogress', monetizationprogressHandler);
+
+    // Coil's web monetization API is down because they have stopped their service.
+    //
+    //if (document['monetization']) {
```

Comment:
I don't think it make sense to keep this code here. 
They stop their service so is not a short time thing and probably even if they start the service again, they will have different code. 
I propose to delete these 3 lines in favour of keep the code clean and less confusing.

Reply:
Done! @gsambrotta